### DISCUSSION TITLE
Toastify integration

### DISCUSSION THREAD

**Discussion by ajafov98** [quasarframework/quasar#15777] [General - Components / Directives / etc] [answered]
I want to add [vue3-toastify](https://www.npmjs.com/package/vue3-toastify)() to my Quasar app (Quasar CLI, Vite).
I tried to add a boot file for Toastify:
```
export default boot(({app}) => {
  app.use(Vue3Toastify, { autoClose: 3000 });
});

```

Then added the boot file name to boot array in quasar config, but no result. Played a bit with boot file but couldn't get any results. 
Is there any way to achieve this?

**Comment by smolinari** [comment]
Out of curiosity, how about using the notify plugin and create your own toasts?

https://codepen.io/smolinari/pen/jOeGYpb?editors=101

Scott

**Reply by ajafov98** [reply]
I really like the Notify plugin, however, I've encountered some difficulties when trying to adjust its dimensions using a custom class. Specifically, the close button position creates a somewhat awkward view when making the toast bigger. That's why I decided to use Toastify. It's close button is on the top-right part of the toast regardless of the size.

**Reply by smolinari** [reply]
Do you mean the "X" top right?

**Reply by ajafov98** [reply]
Yes

**Reply by smolinari** [reply]
@ajafov98 - Looks like I'm not capable enough to make the close button top right. But, I got this far too. :grin: 

https://codepen.io/smolinari/pen/jOeGYpb?editors=1010

Scott

**Reply by ajafov98** [reply]
I tried your example with a multi-line message, and the 'x' button actually moved down 😅😅😅

**Reply by smolinari** [reply]
This is closer, but yeah. Not really perfect either. 

https://codepen.io/smolinari/pen/jOeGYpb

Scott

**Comment by ajafov98** [comment, marked as answer]
Just wanted to give an update on this issue - turns out it was my bad. I didn't import the CSS class for Toastify plugin in the boot file. Once I added it, everything worked like a charm.
